.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+module.exports = {
```

Comment:
I've added some linting rules to the `tslint` file. How will they interact with this file?